src/components/page/Header/index.tsx

```diff
@@ -255,6 +260,17 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
     );
   }
 
+  private handleLogout() {
+    const logoutURL: string = 'https://reveal-stage.smartregister.org/opensrp/logout.do';
```

Comment:
lets put this in the env file,

Reply:
we will indeed.

Reply:
> lets put this in the env file

agreed, let's pipe in in from `.env` through `env.ts` and import it here, and let's add it to `.env.sample` too